lib/matplotlib/axes/_axes.py

```diff
@@ -31,9 +28,12 @@
 import matplotlib.ticker as mticker
 import matplotlib.transforms as mtransforms
 import matplotlib.tri as mtri
-from matplotlib.container import BarContainer, ErrorbarContainer, StemContainer
+import numpy as np
```

Comment:
please leave the numpy imports at the top (between the stdlib imports and the matplotlib imports).